components/metrics/RealtimeLog.js

```diff
@@ -137,7 +137,11 @@ export default function RealtimeLog({ data, websites, websiteId }) {
             ),
             browser: <b>{BROWSERS[browser]}</b>,
             os: <b>{os}</b>,
-            device: <b>{intl.formatMessage(devices[device])?.toLowerCase()}</b>,
+            device: <b>
+                {devices[device]
+                  ? intl.formatMessage(devices[device]).toLowerCase()
+                  : intl.formatMessage({ id: 'label.unknown', defaultMessage: 'Unknown' })}
```

Comment:
You should pull the unknown message from `components/messages.js`.

Reply:
Actually, there is a `getDeviceMessage` method in that file, use that instead.

Reply:
@mikecao Thank you for your review.
I've fixed it, so please review it again:pray: